source/addonHandler/addonVersionCheck.py

```diff
@@ -18,6 +20,15 @@ def getAddonCompatibilityMessage() -> str:
 	)
 
 
+def getAddonCompatibilityConfirmationMessage() -> str:
+	return pgettext(
+		"addonStore",
+		# Translators: A message to confirm that the user understands that incompatible add-ons
+		# will be disabled after installation, and can be manually re-enabled.
+		"I understand that incompatible add-ons will be disabled and can be manually re-enabled at my own risk."

```

Comment:
Perhaps add "later" or "after installation" to the end of that sentence.
"I understand that incompatible add-ons will be disabled and can be manually re-enabled at my own risk after installation."